repo-agent/images/issue-sandbox/Dockerfile

```diff
@@ -1,5 +1,21 @@
-# Build stage
+# Build the issue-sandbox binary
+FROM golang:1.24 as builder
+WORKDIR /app
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the llm directory
+COPY pkg/llm/ ./pkg/llm/
+
+COPY issue-sandbox/ .
+RUN CGO_ENABLED=0 GOOS=linux go build -o /issue-sandbox .
```

Comment:
nit: I believe we can add -ldflags="-s -w" to the go build command to make the binary a bit smaller.  IIUC this strips debugging information and symbols which likely aren't needed.

Reply:
done